isabelle/Doc/Specification/Core.thy

```diff
@@ -176,13 +171,8 @@ text \<open>Division is a special case because we only evaluate to natural numbe
 
 text \<open>@{thm [display,names_short, margin=40] evalValue_DivValue}\<close>
 
-(* \<open>TODO: lemmas around division? maybe extend the following to proof evalValue and not just div\<close>*)
-text \<open>@{thm divMultiply}\<close>
-text \<open>@{thm divAbsMultiply}\<close>
-(*text \<open>COMMENT(BWB): I suggest that the lemmas be (i) exact multiples divide with no remainder, (ii)
```

Comment:
@hrajchert: Shouldn't we add the suggested lemmas as well?

Reply:
Not necessarily in the specification PDF. Those lemmas might be needed later if we want to prove some rewrite rules.